src/SFA.DAS.EmployerAccounts/Queries/GetLastSignedAgreement/GetLastSignedAgreementQueryValidator.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Threading.Tasks;
+using SFA.DAS.Validation;
+
+namespace SFA.DAS.EmployerAccounts.Queries.GetLastSignedAgreement
+{
+    public class GetLastSignedAgreementQueryValidator : IValidator<GetLastSignedAgreementRequest>
+    {
+        public GetLastSignedAgreementQueryValidator()
+        {
+        }
+
+        public ValidationResult Validate(GetLastSignedAgreementRequest item)
+        {
+            throw new NotImplementedException();
```

Comment:
can we just call the async method and return that synchronously rather than just throw?